python/lsst/meas/base/plugins.py

```diff
@@ -371,6 +372,38 @@ def fail(self, measRecord, error=None):
         pass
 
 
+class SingleFrameFootprintAreaConfig(SingleFramePluginConfig):
+    pass
+
+
+@register("base_FootprintArea")
+class SingleFrameFootprintAreaPlugin(SingleFramePlugin):
+    """A measurement plugin that simply records the area of the source's
+    detection Footprint.
+    """
+
+    def __init__(self, config, name, schema, metadata):
+        SingleFramePlugin.__init__(self, config, name, schema, metadata)
+        self.key = schema.addField(
+            schema.join(name, "value"),
+            type=numpy.int32,
+            doc="Number of pixels in the source's detection footprint.",
+            units="pixel"
+        )
+
+    @classmethod
+    def getExecutionOrder(cls):
+        return cls.CENTROID_ORDER  # probably doesn't need to go early, but can
```

Comment:
I don't think we are supposed to do comments after statements. Additionally the wording of the comment is a bit awkward, and required re-reading to notice it was not cut short somehow